docs/src/concepts/bc.md

```diff
@@ -23,6 +21,27 @@ Followingly, we describe the syntax in [Chmy.jl](https://github.com/PTsolvers/Ch
 | Dirichlet | `bc!(arch, grid, field => Dirichlet())` | `bc!(arch, grid, field => Dirichlet(v))` |
 | Neumann | `bc!(arch, grid, field => Neumann())` | `bc!(arch, grid, field => Neumann(v))` |
 
+
+### Batched Boundary Conditions
+
+For better performance considerations, one could also combine the application of boundary conditions with kernels dedicated to field updates by using batched boundary conditions.
+
+Given a 2D vector field `V`, we specify boundary conditions imposed on `V.x` and `V.y` as a tuple `bc_V`.
+
+```julia
+# specify for both V.x and V.y fields
+bc_V = (V.x => (x=Dirichlet(), y=Neumann()),
+        V.y => (x=Neumann(), y=Dirichlet()))
+```
+
+When using `launch` to specify the execution of a kernel, one can pass the specified boundary conditions as an optional parameter using `batch`, provided the grid information of the discretized space.
+
+```julia
+launch(arch, grid, update_velocity! => (V, r_V, Pr, τ, ρgy, η_ve, νdτ, grid); bc=batch(grid, bc_V...))
+```
+
+Using this syntax, we avoid redundant kernel launches and gain efficiency from making good use of already cached values used in field updates.
```

Comment:
This may not be accurate. Even if we have the batched approach, we still launch a kernel in order to apply the BCs.

Reply:
removed inaccurate description in this [commit](https://github.com/PTsolvers/Chmy.jl/pull/24/commits/503237ca015f57886d1778b572e12dcc212fbb0e)